pkg/api/validation/validation.go

```diff
@@ -2844,10 +2799,16 @@ func ValidateNodeUpdate(node, oldNode *api.Node) field.ErrorList {
 	// Clear status
 	oldNode.Status = node.Status
 
+	// update taints
```

Comment:
@derekwaynecarr Yes - I think saying you can add a toleration but not change an existing one is very reasonable (both for this PR and long-term). @liggitt would it be OK for us to do that in this PR?